simulation_node:
  ros__parameters:
    rock_structure_mesh:
      mesh: '/home/akshay/mesh_files/closed_pbr.obj'
      meshScale: [1.0, 1.0, 1.0]
      mass: 1437.596
      restitution: 0.3
      lateralFriction: 0.5
      spinningFriction: 0.3
      rollingFriction: 0.3
      contactDamping: 1.0
      contactStiffness: 100000.0
